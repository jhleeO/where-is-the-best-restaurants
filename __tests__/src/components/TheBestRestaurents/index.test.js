
import 'react-native';
import React from 'react'
import renderer from 'react-test-renderer';


describe('TheBestRestaurents', () => {
  let TheBestRestaurents;

  beforeEach(() => {
    TheBestRestaurents = require('../../../../src/components/TheBestRestaurents').default;
  })

  it('Randering Possible.', () => {
    //given

    //when
    const json = renderer.create(<TheBestRestaurents/>).toJSON();
    //then
    expect(json).toMatchSnapshot();
  });

});

