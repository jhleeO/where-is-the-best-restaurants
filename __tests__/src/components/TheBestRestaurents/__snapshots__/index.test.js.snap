// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheBestRestaurents Randering Possible. 1`] = `
<View
  style={
    Object {
      "backgroundColor": "yellow",
      "height": 1084,
      "marginTop": 50,
      "width": 750,
    }
  }
/>
`;
